src/main/java/io/jenkins/plugins/customizable_header/UserHeader.java

```diff
@@ -20,12 +25,13 @@
 
 public class UserHeader extends UserProperty {
 
-  private boolean overwriteHeader;
+  private boolean enabled;
+
+  private boolean thinHeader;
 
-  private boolean overwriteColors;
   private HeaderColor headerColor;
 
-  private HeaderSelector headerSelector;
+  private Logo logo;
```

Comment:
I don't think it is a good idea to let users define the logo. A regular user typically doesn't know what a symbol is, he might not have the possibility to upload svg files to the userContent folder.